release/nightly_tests/placement_group_tests/cluster.py

```diff
@@ -0,0 +1,13 @@
+from ray.cluster_utils import Cluster
```

Comment:
This file is needed for the local repro. I will make issues for local repro

Reply:
Thank you for adding the repro!